### PR TITLE
[tools] Continue on pcd load failure in train_linemod

### PR DESCRIPTION
**Pull request by kunaltyagi**

### CODE REVIEW

File: tools/train_linemod_template.cpp [hunk 1]
```diff
@@ -299,4 +306,12 @@ main (int argc, char** argv)
...
   }
 
+  if (error_detected)
+  {
+    return -1;
+  }
+  else
+  {
+    return 0;
+  }
```
Comment (SergioRAgostinho) [inline tools/train_linemod_template.cpp]: Return codes from applications are usually non-negative integers. In this situation I would literally just do
```cpp
return error_detected;
```
I leave it you the decision of the (non)explicit cast.

Comment (kunaltyagi) [reply tools/train_linemod_template.cpp]: PTAL

Comment (SergioRAgostinho) [reply tools/train_linemod_template.cpp]: :+1:  It's still returning a negative integer though. Any reason for that choice?

Comment (kunaltyagi) [reply tools/train_linemod_template.cpp]: It was returning a [negative integer](https://github.com/PointCloudLibrary/pcl/pull/3652/files#diff-66720143805e877dec9fbeaa90a0326cL270) before. I kept the same behavior. No other rationale.

**Pull request merged**